README.md

```diff
@@ -36,6 +36,12 @@ Docker Compose installation options.
 
 (Used only for RedHat/CentOS.) You can enable the Edge or Test repo by setting the respective vars to `1`.
 
+Set the docker_users variable to allow those users to run docker (these users are added to the docker group)
```

Comment:
This comment should appear below the variable (to follow the convention with the other vars in the README).